examples/audio_out.c

```diff
@@ -348,7 +348,13 @@ opensoundsys_open (int channels, int samplerate)
 		exit (1) ;
 		} ;
 
-	fmt = CPU_IS_BIG_ENDIAN ? AFMT_S16_BE : AFMT_S16_LE ;
+	fmt =
+#ifdef WORDS_BIGENDIAN
+	AFMT_S16_BE
+#else
+	AFMT_S16_LE
+#endif
```

Comment:
1. is only a solution right now it is not a long term solution.

I can however live with solution 2.